models/opx_finance/optimism/opx_finance_optimism_trades.sql

```diff
@@ -0,0 +1,103 @@
+{{ config(tags=['dunesql'],
+    aliAS = alias('trades')
```

Comment:
minor: can you lowercase `alias` here? also, maybe move the comma to left side similar to below, to make this model config quick/easier to look at in future?

Reply:
Ok

Reply:
Done. Kindly recheck.